app/scripts/modules/core/application/modal/editApplication.controller.modal.js

```diff
@@ -19,6 +20,7 @@ module.exports = angular
     vm.errorMsgs = [];
     vm.application = application;
     vm.applicationAttributes = _.cloneDeep(application.attributes);
+    vm.applicationAttributes.instancePort = vm.applicationAttributes.instancePort || settings.defaultInstancePort || 8080;
```

Comment:
Whoops, good catch. It should probably default to port 80 if nothing's configured.